src/shared/dropdown-list/dropdown-list.tsx

```diff
@@ -227,6 +232,14 @@ export const DropdownList = <T, V>({
         return listHeight + customCallToActionHeight;
     };
 
+    const hasTwoLinesLabel = () => {
```

Comment:
the label handling can be streamlined, the combined handling of `itemTruncationType` and `itemFlexDirection` is confusing at the moment

I'm thinking `itemFlexDirection` can be reworked to a `secondaryLabelDisplayType: "inline" | "next-line"` to cater for other possible variants

dropdown-list can render the label as:
```
<Label $secondaryLabelDisplayType={secondaryLabelDisplayType}>
  <Title $truncateType={...}>
    {itemTruncationType === "middle" ? renderTruncatedText(title) : title}
  <Title>
  {secondaryLabel &&
    <Description $truncateType={...}>
      {itemTruncationType === "middle" ? renderTruncatedText(secondaryLabel) : secondaryLabel}
    </Description>}
</Label>
```

Reply:
realised the current component in prod looks wonky when the title and label are both long. have amended the implementation so that the text is at least visible. still pending confirmation from UX on the truncation handling for inline labels.